test/src/unit-capi-vfs.cc

```diff
@@ -322,6 +322,9 @@ void VFSFx::check_vfs(const std::string& path) {
   // Move
   check_move(path);
 
+  // Copy
+  check_copy(path);
```

Comment:
This still gets called on Windows, so the test fails looking for the symbol/implementation. Wrap this is in an `#ifndef _WIN32` as well.